### PR TITLE
Add `receive_nothing` to the docs

### PR DESCRIPTION
**Pull request by dgilge**
Documentation for PR https://github.com/django/asgiref/pull/37

### CODE REVIEW

File: docs/topics/testing.rst [hunk 1]
```diff
@@ -61,21 +65,59 @@ To construct it, pass it an application and a scope::
...
+    assert event.get("more_body") is None
+    # Check that there isn't another event
+    assert await communicator.receive_nothing() is True
+    # You could continue to send and receive events
+    # await communicator.send_input(...)
+
+The method has two optional parameters:
+
+* ``timeout`` is ``0.1`` by default to keep tests short when there is no event
+  to receive and should be more than enough time.
```
Comment (andrewgodwin) [inline docs/topics/testing.rst]: I'd rephrase this a bit. Something like `timeout: number of seconds to wait to ensure the queue is empty. Defaults to 0.1.`

File: docs/topics/testing.rst [hunk 2]
```diff
@@ -61,21 +65,59 @@ To construct it, pass it an application and a scope::
...
+* ``interval`` is the time between two checks for new events. (It will return after
+  a check if there is an event.) The default of ``0.01`` should work well. However,
+  if you want to customize it you might use a low value (``0.001``) if you expect an
+  event and a high value (equal to ``timeout``) if you don't.
```
Comment (andrewgodwin) [inline docs/topics/testing.rst]: Mention interval is in seconds as well, and drop commentary about how well the default works - just say the default is `0.01` and let people who need to tweak it do so. We don't want to have most people worrying about this value.

**Comment by dgilge** [comment]
I shortened the parameter explanations.

**Comment by andrewgodwin** [comment]
Thanks!
